lib/fathead/python/cover/functions.txt

```diff
@@ -106,3 +106,13 @@ def
 for
 lambda
 try
+csv read, csv.reader
```

Comment:
@mustang25 these redirects need to be placed in a new file, `redirects.txt` placed in `lib/fathead/python/`